backend/app/api/shortcuts.py

```diff
@@ -0,0 +1,23 @@
+from fastapi import HTTPException, status
```

Comment:
IMO, I think it fits better in the `api` folder since it relies on FastAPI dependency injection scheme and it raises an `HTTPException`. It seems really tied to the API/HTTP to me.